requirements-dev.txt

```diff
@@ -6,4 +6,5 @@ pybind11
 pytest
 pytest-cov
 pytest-mock
-black==23.7.0
\ No newline at end of file
+black==23.7.0
+clang-format==14
```

Comment:
Do we need that? Or maybe you forgot it here?

Reply:
I think it's good to have it since these are the versions used by the CI and pre-commit hooks.